src/stories/theme/_list.css

```diff
@@ -1,3 +1,41 @@
+/**
+ * [1] - Due to PostCSS limitations in CKEditor we must include our icon
+ *       directly.
+ */
+.ck.ck-list__item {
```

Comment:
Per designs, `:first-child` and `:last-child` should have a 6px margin top/bottom respectively.

Reply:
I've updated this styling with the `_dropdown` customizations since lists are used in other areas than toolbar dropdowns.